lib/src/disposable/disposable.dart

```diff
@@ -14,6 +14,7 @@
 
 import 'dart:async';
 import 'dart:collection';
+import 'dart:html';
```

Comment:
Tomorrow let's look at how we might be able to remove this dependency, or at least mitigate it.